nemo/collections/nlp/modules/common/megatron/module.py

```diff
@@ -34,6 +45,10 @@ class MegatronModule(torch.nn.Module):
 
     def __init__(self, share_word_embeddings=True):
         super(MegatronModule, self).__init__()
+
+        if not HAVE_APEX:
+            logging.warning("Apex was not found. Using model parallel or megatron models will error out.")
```

Comment:
Should we leave this as a warning or just ask the user to install Apex here?

Reply:
Updated the message.